support-internationalisation/README.md

```diff
@@ -10,7 +10,10 @@ Run `sbt +publishLocal`.
 Releasing to maven
```

Comment:
I've updated the readme on support-internationalisation as well because it was pretty terse before!